ui-kit/src/main/kotlin/org/p2p/uikit/organisms/seedphrase/SecretKey.kt

```diff
@@ -0,0 +1,6 @@
+package org.p2p.uikit.organisms.seedphrase
+
+data class SecretKey(
```

Comment:
SecretKey -> SeedPhraseWord

Reply:
`SeedPhraseKey` mb?

Reply:
Well it's not kind of key. Let's deep dive into the domain.
So the secret key a.k.a. master key - is the key that is used in Solana for user public key and private key generation. 
I suggested to rename all `Secret` to remove ambiguous sense. 

So we have a seed phrase. What is a phrase? It's a set of words, right? 
`hello main qwerty azerty` - seed phrase
`hello` one of the words in the seed phrase

A seed phrase doesn't have any Keys inside, really. We can't use it as a map. 
That's why a `Word` is better here

Reply:
Okay, no problem.
I just don't like the word `Word` and that's it!)

Reply:
do you hate Microsoft Word too?  🤣

Reply:
I mean in terms of code it looks unprofessional or smth weird 😄